### PR TITLE
Implemented the help dropdown

### PR DESCRIPTION
**Pull request by rodrigopavezi**

### CODE REVIEW

**Comment by rodrigopavezi** [comment]
@ezequielc could you please review? cheers

File: bower.json [hunk 1]
```diff
@@ -33,10 +33,13 @@
     "angular-ui-flow-manager": "https://github.com/Rise-Vision/angular-ui-flow-manager.git",
     "ng-gapi-loader": "https://github.com/Rise-Vision/ng-gapi-loader.git",
     "ng-core-api-client": "https://github.com/Rise-Vision/ng-core-api-client.git",
-    "rv-common-style": "https://github.com/Rise-Vision/common-style.git",
+    "rv-common-style": "https://github.com/Rise-Vision/common-style.git#1.3.19",
     "rv-loading": "https://github.com/Rise-Vision/rv-loading.git",
     "rv-common-app-components": "https://github.com/Rise-Vision/common-app-components.git",
     "rv-common-svg": "https://github.com/Rise-Vision/common-svg.git",
+    "component-subscription-status": "https://github.com/Rise-Vision/component-subscription-status.git#master",
+    "widget-settings-ui-core": "https://github.com/Rise-Vision/widget-settings-ui-core.git",
```
Comment (ezequielc) [inline bower.json]: @rodrigopavezi Where is this dependency being used?

Comment (rodrigopavezi) [inline bower.json]: @ezequielc it is being used on the support-factory in here https://github.com/Rise-Vision/common-header/pull/589/files#diff-06f638e9f862375e0b1a5b896666630bR4. cheers

File: src/templates/help-dropdown.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      <ng-include
+              replace-include
+              src="'help-dropdown-menu.html'">
+      </ng-include>
+  </div>
+</li>
+
+<!-- Mobile -->
+<li ng-show="!isLoggedIn || (isLoggedIn && isRiseVisionUser)" class="visible-xs-inline-block" >
+    <a href="" class="company-buttons-icon-mobile"
```
Comment (ezequielc) [inline src/templates/help-dropdown.html]: @rodrigopavezi Rendering as mobile shows the wrong icon. It should be the question mark. Cheers

Comment (rodrigopavezi) [inline src/templates/help-dropdown.html]: @ezequielc good catch, I will change it. cheers

File: bower.json (hunk 1 above)
Comment (rodrigopavezi) [inline bower.json]: @ezequielc you are right, the widget-settings-ui-core comes in automatically. I will remove it. cheers

**Comment by rodrigopavezi** [comment]
@ezequielc I have made the changes, please review. cheers

**Comment by ezequielc** [comment]
@rodrigopavezi Looks good! :+1:

**Pull request merged**